packages/fgtclb/academic-persons-edit/Classes/Domain/Model/Dto/AbstractFormData.php

```diff
@@ -24,4 +30,51 @@ public function _hasProperty(string $propertyName): bool
     {
         return property_exists($this, $propertyName);
     }
+
+    final public function setRequest(ServerRequestInterface $request): void
+    {
+        $this->request = $request;
+    }
+
+    final protected function getRequest(): ?ServerRequestInterface
+    {
+        return $this->request;
+    }
+
+    final public function setArgumentName(string $argumentName): void
+    {
+        $this->argumentName = $argumentName;
+    }
+
+    final public function getArgumentName(): ?string
+    {
+        return $this->argumentName;
+    }
+
+    protected function getExtbaseRequestParameters(): ?ExtbaseRequestParameters
+    {
+        return $this->request?->getAttribute('extbase') ?? null;
+    }
+
+    /**
+     * @return array<string, mixed>|null
+     */
+    protected function getActionRequestArguments(): ?array
+    {
+        return $this->getExtbaseRequestParameters()?->getArguments() ?? null;
+    }
+
+    public function wasPropertySendInRequest(string $propertyName): bool
```

Comment:
```suggestion
    public function wasPropertySentInRequest(string $propertyName): bool
```

Reply:
Merci. Will adopt that when continuing on this change. If kind of naming will end up in still matching ;)